test/e2e/network/loadbalancer.go

```diff
@@ -1195,18 +1195,11 @@ var _ = common.SIGDescribe("ESIPP [Slow]", func() {
 
 		// Poll till kube-proxy re-adds the MASQUERADE rule on the node.
 		ginkgo.By(fmt.Sprintf("checking source ip is NOT preserved through loadbalancer %v", ingressIP))
-		var clientIP string
-		pollErr := wait.PollImmediate(framework.Poll, e2eservice.KubeProxyLagTimeout, func() (bool, error) {
-			clientIP, err := GetHTTPContent(ingressIP, svcTCPPort, e2eservice.KubeProxyLagTimeout, "/clientip")
-			if err != nil {
-				return false, nil
-			}
-			if strings.HasPrefix(clientIP, "10.") {
-				return true, nil
-			}
-			return false, nil
-		})
-		if pollErr != nil {
+		clientIP, err := GetHTTPContent(ingressIP, svcTCPPort, e2eservice.KubeProxyLagTimeout, path)
```

Comment:
Are we sure this is what we want to do? Don't we need to handle the case where the proxy has not yet applied to the change, the HTTP request goes through, but it looks wrong?

Reply:
it does check it in Line1202, but my reading is that we shouldn't have a reason to receive a client IP that is no from the 10.x.x.x network

Reply:
ahh, I see, it is an update, I didn't see it before, you are right

Reply:
I've recovered it, the outer loop should have a longer period than the inner loop though